src/main/java/ch/njol/skript/effects/EffCharge.java

```diff
@@ -49,10 +55,17 @@ protected void execute(Event event) {
 				creeper.setPowered(charge);
 			} else if (entity instanceof WitherSkull witherSkull) {
 				witherSkull.setCharged(charge);
-            }
+            } else {
+				warning("An entity passed to the 'charge' effect was not a creeper, nor a wither skull, and was thus unaffected.", entities.toString());
+			}
```

Comment:
again, i don't think this needs to exist
if you give it an entity that can't be charged, then it's not going to be charged. That's expected normal behavior.
plus the indentation is messed up and this would spit out 1 warning for each entity, each run. That could be a lot of warnings.